src/elements/link-list-anchor/readme.md

```diff
@@ -0,0 +1,52 @@
+The `sbb-link-list-anchor` is a component that can be used to collect and display more [sbb-block-link](/docs/elements-sbb-link-sbb-block-link--docs).
```

Comment:
or similiar
```suggestion
The `sbb-link-list-anchor` is a component that can be used to collect and display a couple of [sbb-block-link](/docs/elements-sbb-link-sbb-block-link--docs).
```

Reply:
I suggest (I also don't like 'more' and 'a couple' seems a bit too imprecise)
"The `sbb-link-list-anchor` is a component that can be used to collect and display [sbb-block-link](/docs/elements-sbb-link-sbb-block-link--docs)."

Reply:
Also ok for me :)